app/views/posts/show.html.erb

```diff
@@ -12,4 +12,9 @@
       <li><%= comment.user.name %> : <%= comment.text %></li>
     <% end %>
   </ul>
+  <%= form_with url: "/posts/#{@post.id}/likes", scope: :like do |form| %>
+  <%= form.submit "Like" %>
+<% end %>
+
+ <%= render partial: "create_comments", local: { post: @post } %> 
 </div>
```

Comment:
- [ ] I notice that I can't have access to the Post#Show page from the UI (no button or link to lead to it) which eans I can't access the comments form from the UI. I have to manually insert the URL in the browser pane. This is not a good user experience because users won't know the correct URL to fix. Kindly implement this change. 👍 💻